assets/messages.yml

```diff
@@ -1039,6 +1039,19 @@ messages:
       shortcut: qul
       message: "%quick_links%"
       fillname: []
+  removed-version:
+    - project:
+        - mc
+        - mce
+        - mcl
+        - mcpe
+      name: Panel - Removed Extra Version
+      shortcut: remver
+      message:
```

Comment:
Maybe it would be better to use a YAML literal block scaler (i.e. `|- ...`) here.